src/ray/core_worker/object_interface.cc

```diff
@@ -1,4 +1,5 @@
 #include "ray/core_worker/object_interface.h"
+#include <algorithm>
```

Comment:
there should be an empty line after `#include "ray/core_worker/object_interface.h"` and another one after `#include <algorithm>` (see the google style guide)

Reply:
thanks